helper.js

```diff
@@ -1311,7 +1311,7 @@ if (typeof module !== "undefined" && module.hasOwnProperty("exports")) {
 }
 
 if (typeof define !== "undefined") {
-	define([], function () {
+	define(["toBlob"], function (toBlob) {
```

Comment:
Also I am not sure we should require it in helper. It is after all a polyFill and could just be required in the app.